Dockerfile

```
@@ -24,7 +27,6 @@ COPY --from=builder /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver/bin/ibm-pow
 ENTRYPOINT ["/ibm-powervs-block-csi-driver"]
 
 # centos base image
-FROM quay.io/centos/centos:stream8 AS centos-base
-RUN yum install -y util-linux nfs-utils e2fsprogs xfsprogs ca-certificates && yum clean all && rm -rf /var/cache/yum
+FROM --platform=$TARGETPLATFORM ghcr.io/kubernetes-sigs/ibm-powervs-block-csi-driver-centos-base:v8.0.0 AS centos-base
```

seems like there are some security concerns about making GH docker image public, meanwhile wondering if following change might help us improve some build timing

```suggestion
FROM --platform=$TARGETPLATFORM quay.io/centos/centos:stream8 AS centos-base
RUN yum install -y util-linux nfs-utils e2fsprogs xfsprogs ca-certificates && yum clean all && rm -rf /var/cache/yum
```

---

Sure I will test with the changes.